Documentation/network/servicemesh/default-deny-ingress-policy.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_default_deny_ingress_policy:
+
+***************************
+Default Deny Ingress Policy
+***************************
+
+Let's apply clusterwide network policy to deny all the traffic by default.
+
+.. literalinclude:: ../../../examples/kubernetes/servicemesh/policy/default-deny.yaml
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/policy/default-deny.yaml
+
+With this policy applied, the request to ``/details`` endpoint will be denied for external and in-cluster traffic.
```

Comment:
RST uses single back-ticks for literals, rather than how markdown does it with double back-ticks. Can we also add a lead-in sentence here, similar to how you did below for internal traffic?

```suggestion
With this policy applied, the request to the `/details` endpoint will be denied for external and in-cluster traffic.

Let's check if external traffic is denied:
```